README.md

```diff
@@ -1,6 +1,8 @@
 # Bugsnag exception reporter for Android
 [![Documentation](https://img.shields.io/badge/documentation-latest-blue.svg)](http://docs.bugsnag.com/platforms/android/)
 [![Build status](https://travis-ci.org/bugsnag/bugsnag-android.svg?branch=master)](https://travis-ci.org/bugsnag/bugsnag-android)
+<a href="http://www.methodscount.com/?lib=com.bugsnag%3Abugsnag-android%3A3.9.0"><img src="https://img.shields.io/badge/Methods and size-core: 661 | deps: 32 | 73 KB-e91e63.svg"/></a>
```

Comment:
The output looks great, can you switch the formatting to use markdown like the links above? I believe shields.io supports using underscores or something to represent spaces.